src/components/Books/BookForm.jsx

```diff
@@ -1,14 +1,51 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../../redux/books/books';
 
 function BookForm({ categories }) {
+  const dispatch = useDispatch();
+
+  const [formState, setFormState] = useState({});
+
+  const resetForm = (e) => {
+    const form = e.target;
+    Array.from(form.querySelectorAll('input')).forEach(
+      // eslint-disable-next-line no-param-reassign
```

Comment:
[OPTIONAL] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.

Reply:
I searched for a better way to achieve this without disabling the linters. Can you suggest a better way?